typescript/rds/oracle/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "oralce",
+  "version": "0.1.0",
+  "bin": {
+    "cdk": "oralce.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.10",
+    "@types/node": "*",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.0.0",
+    "constructs": "^10.0.0",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "source-map-support": "^0.5.16"
+  }
```

```suggestion
  "devDependencies": {
    "@types/jest": "^26.0.10",
    "@types/node": "*",
     "jest": "^26.4.2",
    "ts-jest": "^26.2.0",
    "ts-node": "^9.0.0",
    "typescript": "~3.9.7"
  },
  "dependencies": {
    "aws-cdk-lib": "^2.0.0",
    "constructs": "^10.0.0",
    "source-map-support": "^0.5.16"
  }
```

Same as other package.json